extensions/amp-digidip/validator-amp-digidip.protoascii

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-digidip
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-digidip"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-digidip>
+  html_format: AMP
+  tag_name: "AMP-DIGIDIP"
+  requires_extension: "amp-digidip"
+  attrs: {
```

Comment:
Are there any restrictions that these attributes should have on their values? These can be enforced via explicitly naming the values or using a regex.